company/values.md

```diff
@@ -27,7 +28,30 @@ We prefer to [build teams, not workgroups](https://dzone.com/articles/workgroups
 
 As a team's size and scope grows, it (unintentionally) tends to become a workgroup. Resist this by aggressively focusing, or by splitting it into smaller teams.
 
-## Open and transparent
+### High agency
+
+Everyone at Sourcegraph has the power and the responsibility to improve Sourcegraph as a company and as a product.
+
+> “Life can be much broader once you discover one simple fact: Everything around you that you call life was made up by people that were no smarter than you and you can change it, you can influence it, you can build your own things that other people can use. Once you learn that, you'll never be the same again.” - Steve Jobs
+
+A few examples of how we encourage high agency are by being a [handbook-first](https://about.sourcegraph.com/handbook/usage#why-handbook-first) company, which allows anyone to change how the business operates, and by empowering engineers to spend time on projects they believe in through our policy of [innovation time](https://about.sourcegraph.com/handbook/engineering#innovation-time).
+
+With agency comes responsibility; employees are expected to take initiative and engage in collaboration when exercising their agency.  If you feel that something is important, then advocate for it! If you feel like you're using the wrong tools for the job or doing the wrong thing, then you're responsible for quickly voicing that concern, and engaging with your teammates to find the solution that is the best for the team. This doesn't mean that you should make unilateral decisions. As an [open company](index.md#open-company), we value open discussion on important topics, and think that collaboration should be a large part of making things better. It is not a bad outcome if your proposal is not accepted or replaced with a better solution, because you took the initiative, started the conversation, and either reaffirmed that our current solution is good, or implemented an even better one!
+
+### High quality
+
+Every person on our team is individually responsible for knowing what high-quality work looks like and producing high-quality work. You divide large problems into small low-risk, tractable problems that can be solved iteratively.
+
+- If you don’t know what high quality looks like, it's your responsibility to find out (for example, by asking teammates and stakeholders).
```

Comment:
This elaboration solves some of the concerns with the short version above, but I still think we should make it less prone to misinterpretation (above).

Reply:
I agree that I think this value could be updated to be clearer, but for the purposes of this PR it hasn't actually changed (just moved), so I think we should defer editing this value to another PR.

Reply:
Addressed in https://github.com/sourcegraph/about/pull/1760